core/unit_test/TestGraph.hpp

```diff
@@ -739,4 +739,61 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), end_of_submit_control_flow) {
             value_A + 2 * value_B + value_C + value_D + value_E + value_F);
 }
 
+template <typename Exec>
+struct GraphNodeTypes {
+  // Type of a kernel node built using a Kokkos parallel construct.
+  using kernel_t =
+      Kokkos::Impl::GraphNodeKernelImpl<Exec, Kokkos::RangePolicy<Exec>,
+                                        CountTestFunctor<Exec>,
+                                        Kokkos::ParallelForTag>;
+
+  // Type of an aggregate node.
+  using aggregate_t = typename Kokkos::Impl::GraphImpl<Exec>::aggregate_impl_t;
+};
+
+TEST(TEST_CATEGORY, is_graph_kernel_v) {
+  using types = GraphNodeTypes<TEST_EXECSPACE>;
+
+  static_assert(Kokkos::Impl::is_graph_kernel_v<types::kernel_t>);
+  static_assert(!Kokkos::Impl::is_graph_kernel_v<types::aggregate_t>);
+}
```

Comment:
There is no point in having a runtime unit test for that one.
```suggestion
constexpr bool test_is_graph_node() {
  using types = GraphNodeTypes<TEST_EXECSPACE>;

  static_assert(Kokkos::Impl::is_graph_kernel_v<types::kernel_t>);
  static_assert(!Kokkos::Impl::is_graph_kernel_v<types::aggregate_t>);
  return true;
}
static_assert(test_is_graph_node());
```

Reply:
Indeed ! Done, I just used `test_is_graph_kernel` instead.